docker/Dockerfile.x86_64-unknown-linux-gnu.centos

```diff
@@ -1,31 +1,3 @@
+# FIXME: this file is currently never used.
+# it should probably be supported for aarch64 hosts later.
```

Comment:
```suggestion
# HACK: this file is currently never used and only exists for usage with `Dockerfile.native.centos`
# it will be supported for aarch64 hosts, see #751 and #975
```